### PR TITLE
enhance(plugin): bootstraps kubectl kubemove plugin

### PR DESCRIPTION
**Pull request by ashishranjan738**
Signed-off-by: Ashish Ranjan <ashishranjan738@localhost.localdomain>

This commit bootstraps kubemove plugin.

### CODE REVIEW

**Comment by ashishranjan738** [comment]
@umamukkara @AmitKumarDas please review this

**Review by umamukkara — APPROVED** [review]

File: pkg/plugin/v1alpha1/kubemove.go [hunk 1]
```diff
@@ -0,0 +1,14 @@
+package v1alpha1
+
+import "github.com/spf13/cobra"
+
+// NewCmdKubeMove returns kubemove root command
+func NewCmdKubeMove() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "kubectl kubemove",
+		Short: "kubemove cli-plugin is used for interacting with kmov operators",
```
Comment (AmitKumarDas) [inline pkg/plugin/v1alpha1/kubemove.go:9]: this should be `kubemove`

File: pkg/plugin/v1alpha1/kubemove.go [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+		Long:  "kubemove cli-plugin is used for interacting with kmov operators",
```
Comment (AmitKumarDas) [inline pkg/plugin/v1alpha1/kubemove.go:10]: same here

File: pkg/plugin/v1alpha1/version.go [hunk 3]
```diff
@@ -0,0 +1,29 @@
+package v1alpha1
+
+import (
+	"fmt"
+	"runtime"
+
+	"github.com/spf13/cobra"
+)
+
+// NewCmdVersion returns kmov version command
```
Comment (AmitKumarDas) [inline pkg/plugin/v1alpha1/version.go:10]: `kmov` vs. `kubemove`

File: cmd/kubemove/main.go [hunk 4]
```diff
@@ -0,0 +1,19 @@
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/pflag"
+
+	"github.com/kubemove/kubemove/pkg/plugin/v1alpha1"
```
Comment (AmitKumarDas) [inline cmd/kubemove/main.go:8]: can we alias this as `plugin`

File: pkg/plugin/v1alpha1/version.go [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+func NewCmdVersion() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Prints version and other details relevant to kubemove plugin",
+		Long: `Prints version and other details relevant to kubemove plugin
+
+Usage:
+kubectl kubemove version
```
Comment (AmitKumarDas) [inline pkg/plugin/v1alpha1/version.go:18]: can we shorten this?
`kubectl km version`
can both kubemove as well as km be supported when used as a plugin?
e.g.
`kubectl km version`
`kubectl kubemove version`

Comment (ashishranjan738) [reply pkg/plugin/v1alpha1/version.go:18]: yes using the symobolic link

**Pull request merged**